services/print-pdf/_tools/xlsx_to_html_vet_patrol.py

```
@@ -0,0 +1,100 @@
+"""One-shot generator: xlsx vet_patrol_template → HTML grid that mirrors layout.
+
+Run once, paste output into templates/vet_patrol.html. xlsx 不是 runtime 依賴。
+
+Why: reference PDF (動物欄位巡視報告範例.pdf) 是 vet/QA 手調的 spatial 2D layout
+（A/B/C/D/E/F/G 區位置標籤散落、merge cells、空白列保留）。原 xlsx 已是
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)